src/components/entity/ha-entity-picker.ts

```diff
@@ -2,7 +2,7 @@ import { mdiCheck, mdiClose, mdiMenuDown, mdiMenuUp } from "@mdi/js";
 import "@polymer/paper-input/paper-input";
 import "@polymer/paper-item/paper-icon-item";
 import "@polymer/paper-item/paper-item-body";
-import "@vaadin/vaadin-combo-box/theme/material/vaadin-combo-box-light";
+import "@vaadin/combo-box/theme/material/vaadin-combo-box-light";
```

Comment:
In target selector when no entities (not very common use case I know)

![image](https://user-images.githubusercontent.com/1444314/152617047-762c3acd-eee4-4924-a5d6-07857bccf7de.png)